include/swift/AST/DiagnosticsCommon.def

```diff
@@ -44,6 +44,10 @@ NOTE(previous_decldef,none,
 NOTE(brace_stmt_suggest_do,none,
      "did you mean to use a 'do' statement?", ())
 
+WARNING(error_in_future_swift_version,none,
+        "%0; this will be an error in Swift %1",
+        (DiagnosticInfo *, unsigned))
```

Comment:
The diagnostic wrapping approach here is very cool. The "will be" wording feels a little off for existing warnings, because "will be an error in Swift 5" should really be "is an error in Swift 5". Should we say "is an error in Swift 5" instead?

Reply:
Good point. As soon as you use `warnUntilSwiftVersion`, that diagnostic is already an error under that language version. I'll update the wording to your suggestion 👍